### DISCUSSION TITLE
Package Data is All the Way at the Bottom - Mobile Website

### DISCUSSION THREAD

**Discussion by shaunbharat** [npm/feedback#679] [Website] [answered]
When browsing through packages on mobile (website), I have to scroll all the way to the bottom just for the GitHub link or other information.

This can get vey tedious when scrolling past really long README's, so I suggest bringing that information to the top. The GitHub link, homepage and other data just makes more sense to be at the top instead.

**Comment by MylesBorins** [comment, marked as answer]
Thanks for sharing @ShaunB56, we definitely can do a better job here. I'm surfacing to engineering + design to see if we can get this prioritized
